app/cdap/components/DirectiveInput/index.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, IconButton } from '@material-ui/core';
+import InputPanel from 'components/DirectiveInput/Components/InputPanel';
+import DirectiveUsage from 'components/DirectiveInput/Components/DirectiveUsage';
+import {
+  IDirectiveInputProps,
+  IDirectivesList,
+  IDirectiveUsage,
+} from 'components/DirectiveInput/types';
+import { formatDirectiveUsageData, handlePasteDirective } from 'components/DirectiveInput/utils';
+import T from 'i18n-react';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { PREFIX, multipleColumnDirective } from 'components/DirectiveInput/constants';
+import { grey } from '@material-ui/core/colors';
+import CloseOutlinedIcon from '@material-ui/icons/CloseOutlined';
+
+const SimpleBox = styled(Box)`
```

Comment:
Try to make these more contextual rather than using terms like simple

Reply:
Renamed